diff -r 7dd261c59de2 vc/freetype/include/freetype/config/ftconfig.h
--- a/include/freetype/config/ftconfig.h	Mon Jan 08 18:41:57 2018 +0100
+++ b/include/freetype/config/ftconfig.h	Mon Jan 08 18:43:28 2018 +0100
@@ -461,15 +461,6 @@
 #define FT_EXPORT( x )  extern  x
 #endif
 
-#ifdef _MSC_VER
-#undef FT_EXPORT
-#ifdef _DLL
-#define FT_EXPORT( x )  __declspec( dllexport )  x
-#else
-#define FT_EXPORT( x )  __declspec( dllimport )  x
-#endif
-#endif
-
 #endif /* !FT_EXPORT */
 
 
